scripts/generate-docs.sh

```diff
@@ -39,6 +39,9 @@ done
 # Strip extraneous .adoc out of section IDs and xrefs
 sed -i'.bak' -e 's/\.adoc//g' ecctl*.adoc
 
+# Replace ECE availability text for ece icon
+sed -i'.bak' -e 's/(Available for ECE only)/{ece-icon}/g' ecctl*.adoc
```

Comment:
Kudos on doing this for the Asciidoc even through the Markdown doesn't really support it. ❤️

Reply:
🙇‍♀️